go/vt/mysqlctl/version.go

```diff
@@ -36,3 +36,7 @@ func (v *ServerVersion) atLeast(compare ServerVersion) bool {
 	}
 	return false
 }
+
+func (v *ServerVersion) isLTS(compare ServerVersion) bool {
+	return v.Major == compare.Major && v.Minor == compare.Minor
+}
```

Comment:
This would return `true` for 8.3 or 8.5 e.g. and those are not LTS releases ([but "innovation" / development or non-LTS releases](https://dev.mysql.com/blog-archive/introducing-mysql-innovation-and-long-term-support-lts-versions/)). That may be fine in this case. If all we really want to know here is if they are the same major version — which in MySQL is `major.minor` — then we should rename the function to ~ `isSameRelease`.